.gitignore

```diff
@@ -119,7 +119,7 @@ celerybeat.pid
 *.sage.py
 
 # Environments
-.env
```

Comment:
Someone probably has a .env they would like still ignored. Please leave this line in.